lib/features/greeting_screen/widgets/informations_about_app_widget.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:catchqr/theme/app_theme.dart';
```

Comment:
personally I prefer relative imports, but I guess you can discuss tomorrow which one you choose. @mikolaj-jalocha knows how to set up a linter with this rule

Reply:
Yeap, I'd make a taks for setting it up 😋